products/slackware.md

```diff
@@ -0,0 +1,109 @@
+---
+permalink: /slackware
+alternate_urls:
+-   /slackwarelinux
+title: Slackware Linux
+category: os
+releasePolicyLink: http://www.slackware.com/changelog/
+iconSlug: slackware
+changelogTemplate: http://mirrors.slackware.com/slackware/slackware-__RELEASE_CYCLE__/ChangeLog.txt
+activeSupportColumn: false
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+releasePolicyLink: http://www.slackware.com/faq/do_faq.php?faq=general#4
+purls:
+-   purl: pkg:os/slackwarelinux
+auto:
+-   distrowatch: slackware
+    regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "current"
+    codename: "-current"
```

Comment:
```suggestion
    codename: "-current"
    releaseLabel: Current
```

Avoids showing up as "current (-current)"

Reply:
@captn3m0 do we need releaseLabel ? arent all we need is just remove codename tag here ?